lib/cube_browser/__init__.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+
+
+class Contourf(object):
```

Comment:
i'd prefer numpy style docstrings, they're what I'm used to and common across SciTools